RMS/StartCapture.py

```diff
@@ -212,15 +212,23 @@ def runCapture(config, duration=None, video_file=None, nodetect=False, detect_en
     # Initialize buffered capture
     bc = BufferedCapture(sharedArray, startTime, sharedArray2, startTime2, config, video_file=video_file)
 
+    live_view_enable = config.live_view_enable
 
-    # # Initialize the live image viewer
-    # if config.live_view_enable:
+    # Check if platform is ARM/RPi
+    platform = os.uname()[4]

```

Comment:
os.uname() doesn't work on Windows. You could use platform.uname() instead.

Reply:
For now, the capture doesn't works on Windows platform, maybe it's not a problem for now because @dvida is rewriting this module (I think).

Reply:
I'll update code and test soon.

Reply:
Python2.7 (actualy used on RMS) haven't the machine property on result of platform.uname(), on 3.7 have. So, I think we can't use it now.

Reply:
err sorry :(
actually, this is the portable way for the value you want: `platform.machine()`